clio-client/src/main/scala/org/broadinstitute/clio/client/ClioClient.scala

```diff
@@ -247,6 +247,13 @@ class ClioClient(webClient: ClioWebClient,
       _ <- checkRemainingArgs(args ++ args0)
     } yield {
       implicit val bearerToken: OAuth2BearerToken = commonOpts.bearerToken
```

Comment:
A lot of the code uses `OAuth2BearerToken` but it looks like a more general type, `HttpCredentials`, would work instead. That would be preferable to avoid many changes if we ever change the way we do authentication.

Reply:
Also @tlangs 's changes I think

Reply:
It's some weirdness with the commit history, we can fix it up when we go through this to resolve the merge conflicts.